src/controller/CommissionerDiscoveryController.h

```diff
@@ -235,6 +235,21 @@ class DLL_EXPORT AppInstallationService
      */
     virtual bool LookupTargetContentApp(uint16_t vendorId, uint16_t productId) = 0;
 
+    /**
+     * @brief
+     *   Called to check app's installation status.
+     *
+     * CdError status is designed for CommissionerDeclaration and should be set by using SetErrorCode() and
+     * sent back to the client as a CDC Message. It is expected that app installation can have following statuses:
```

Comment:
could we review the text of this comment and see if it can be improved to help reading/using the code?

At a first read: 
  - method is named `GetInstallationStatusOfApp` and description si `checks app installation status` ... is it a get or a check?
  - constants for CdError are listed here ... why and why not just leave it as CdError? Are there specific constants that are never returned? Maybe mentioning that `CdError` has a constant of `kNoError` is important, however listing the rest seems extra.
  
  Overall this seems to be a `gets the status previously set by SetErrorCode, typically during commissioning` based on the description. However then why did we not name it `GetErrorCode` ? Why is this `Application installation status`?

Reply:
Yes, your comment make sense. I've updated the method name and comment of the method.

However as CdError contains more errors that are not related to the app installation feature, I've noted only the ones that are necessary for that.